python/ray/autoscaler/kuberay/ray-cluster.complete.yaml

```diff
@@ -8,13 +8,14 @@ metadata:
     # An unique identifier for the head node and workers of this cluster.
   name: raycluster-complete
 spec:
-  rayVersion: '1.11.0'
+  rayVersion: '1.12.0'
   enableInTreeAutoscaling: false
   ######################headGroupSpecs#################################
   # head group template and specs, (perhaps 'group' is not needed in the name)
   headGroupSpec:
     # Kubernetes Service Type, valid values are 'ClusterIP', 'NodePort' and 'LoadBalancer'
     serviceType: ClusterIP
+    #enableIngress: true
```

Comment:
Let's put a space after `#` to be consistent with the rest of the file

Reply:
I left this line in accidentally -- will remove.

Reply:
Deleted stray line.